src/handlers/call-handler.ts

```diff
@@ -9,7 +9,7 @@ export function CallHandler (externalCallHandler: ExternalCallHandler, eventBus:
   function _externalOrError (functionName: string) {
     const hasExternal = externalCallHandler && externalCallHandler[functionName] && isFunction(externalCallHandler[functionName])
     if (hasExternal) {
-      return externalCallHandler[functionName]
+      return externalCallHandler[functionName].bind(externalCallHandler)
```

Comment:
🤔  why do we need to return the externalCallHandler if the functionName isn't defined?

Reply:
I see thanks for the explanation!